Sources/FeatherDatabase/Database/Query/DatabaseQueryInsert.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  File.swift
+//
+//
+//  Created by Tibor Bodecs on 06/01/2024.
+//
+
+import SQLKit
+
+extension Array {
+
+    fileprivate func chunked(into size: Int) -> [[Element]] {
+        stride(
+            from: 0,
+            to: count,
+            by: size
+        )
+        .map {
+            Array(self[$0..<Swift.min($0 + size, count)])
+        }
+    }
```

Comment:
I recommend adding a dependency on `swift-algorithms` and using its `chunks(ofCount:)` method; it works with all Collections (not just Array) and is significantly more efficient than this implementation.